.github/actions/run-nitro-test-node/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Nitro Test Node
+description: 'Checks out the Nitro repository and runs the local test node'
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        repository: OffchainLabs/nitro
+        submodules: 'recursive'
+        path: 'run-nitro-test-node--nitro'
+        # TODO: Update with a stable ref once available
+        ref: 'test-node-prod'
+
+    - name: Start background nitro test-node.bash
+      shell: bash
+      # The reason for the "&" at the end is https://stackoverflow.com/a/72203688
```

Comment:
NIT: I'd describe the reason here directly, and add the link for more context
`Adding & for parallel run`